src/consts.ts

```diff
@@ -71,17 +71,17 @@ export const MEDIUM_FONT_SiZE_MAX_WIDTH = 1600;
 /*
  * Toggle-able services
  */
-export const SERVICES = ['subscan', 'binance_spot'];
```

Comment:
Does it worth hiding page elements that depend on external services too?

Reply:
Thank you! I removed binance_spot logic. 
Subscan will be adjusted in the next updates.